src/main/kotlin/assimp/DefaultIOSystem.kt

```diff
@@ -1,19 +1,21 @@
 package assimp
 
 import java.io.*
+import java.nio.*
+import java.nio.channels.*
 import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.Paths
 
 class DefaultIOSystem : IOSystem {
 
-    override fun exists(pFile: String) = File(pFile).exists()
+    override fun exists(file: String) = File(file).exists()
```

Comment:
file as string here: more methods / just Path.toString() ?

Reply:
`IOSystem.exists` has to take a String as an argument. I don't want to use `Path` here because it takes a lot of effort to implement the `Path` interface on the readFromMemory side without giving any real benefits.